dash/backend/src/modules/exceptions/services/exceptions.service.ts

```diff
@@ -279,27 +276,7 @@ async getExceptionsForIssueIdentifier(issueIdentifier: string, clusterId: number
   }
 
   async syncGatekeeperBlocks(): Promise<void> {
```

Comment:
Why do we have this same function twice? It's in gatekeeper-exception.command as well.
Can the command service not call this?

Reply:
They're both calling the same service, I assume the difference is so that that can have their own error handling, since the command uses consle.logs to make CLI-user friendly output, and this is using the logger